src/schemas/json/project.json

```diff
@@ -94,12 +94,12 @@
 			"default": ""
 		},
 		"code": {
-			"description": "Glob pattern to specify all the code files that needs to be compiled. (data type: string or array with glob pattern(s)). Example: [ \"Folder1\\*.cs\", \"Folder2\\*.cs\" ]",
+			"description": "Glob pattern to specify all the code files to compile. (data type: string or array with glob pattern(s)). Example: [ \"Folder1/*.cs\", \"Folder2/*.cs\" ]",
```

Comment:
I think `code` property was deprecated. /cc @troydai

See the design spec here: https://github.com/aspnet/dnx/issues/1067#issuecomment-74791920